scala/build/maintenance.Dockerfile

```
@@ -119,19 +119,22 @@
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+    python3-venv && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Optionally verify installation
-RUN python3.10 --version && pip3 --version
+RUN python3.13 --version && pip3 --version
 
 # Upgrade pip to the latest version
-# Found existing installation: pip 24.0
-# RUN python3.10 -m pip install --upgrade pip
+RUN python3.13 -m pip install --upgrade pip
```

Hadolint

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)